src/components/Loaders/IconLoader.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { storiesOf } from "@storybook/react"
+import { withKnobs, select } from "@storybook/addon-knobs"
+import IconLoader from "./index"
+const Readme = require("./README.md")
+
+storiesOf("Loaders", module)
+  .addDecorator(withKnobs)
```

Comment:
We don't need this as we're not adding knobs

Reply:
My bad, I was thinking of adding knobs but then found out there was no need. Have removed this now.